OrbitGl/App.cpp

```diff
@@ -272,7 +272,28 @@ void OrbitApp::PostInit() {
 
     auto callback = [&](ProcessManager* process_manager) {
       main_thread_executor_->Schedule([&, process_manager]() {
-        m_ProcessesDataView->SetProcessList(process_manager->GetProcessList());
+        std::vector<ProcessInfo> process_list =
+            process_manager->GetProcessList();
+
+        uint32_t highest_cpu_pid = 0;
+        if (m_ProcessesDataView->GetSelectedProcessId() == 0) {
+          highest_cpu_pid =
```

Comment:
Could we move that logic to the ProcessManager?

Reply:
But then the process with the max CPU usage would be looked for every time the list is received and that's unnecessary.

Reply:
Sorry, I meant having a GetHighestCpuPID method in ProcessManager instead of having the logic in App.cpp.

Reply:
Ah yeah, makes sense, onto it.

Reply:
Actually @dimitry- what do you think? `ProcessManager` seems to effectively be the gRPC client for the process list. Should it contain more logic?

Reply:
This is client specific logic. I do not think we should do it on the server side. Also I still think this is not a final solution and we probably want to start a game from Orbit in which case we won't use this logic because in this case we know which process to select.

Reply:
I didn't mean on the service side, but as Pierric said in `ProcessManager`, which is still on the client.

Reply:
Oh, I see. I guess it could be there. I do not think we should do that in general, maybe the better way is to remember the name of the process from previous session and try to select it if possible. Selecting highest cpu load seems random and unreliable.

Reply:
As discussed, we'll just go with the CPU usage for now as it's very simple.